main.go

```diff
@@ -41,11 +44,48 @@ func main() {
 		"osArch":    osArch,
 	}
 
-	rootCmd := cmd.NewRootCommand(versions, logFile)
+	logW, err := logFile()
+	if err != nil {
+		logrus.Fatal(err)
+	}
 
-	defer logFile.Close()
+	defer logW.Close()
 
-	if err := rootCmd.Execute(); err != nil {
-		logrus.Fatal(err)
+	h, err := os.Hostname()
+	if err != nil {
+		logrus.Debug(err)
+
+		h = "unknown"
 	}
+
+	t := os.Getenv("FURYCTL_MIXPANEL_TOKEN")
+	if t == "" {
+		panic("FURYCTL_MIXPANEL_TOKEN environment variable not set")
```

Comment:
in this case is not terrible but in general do not use panic unless you really need it.
go for a log + os.Exit(1) instead